### ISSUE TITLE
FastAPI/Starlette: Truncate exception message to prevent "event exceeded the permitted size" error from APM Server

### ISSUE THREAD

**Issue by julianogv** [elastic/apm-agent-python#1614]
**Describe the bug**: 
The capture_exception() method is sending the whole exception message to the APM Server and when the message exceeds the maximum size, the APM Server returns the  "event exceeded the permitted size" error. 

I'm sorry if it is not a bug, but I believe that the apm-agent should truncate the message before sending it to the APM Server.

**To Reproduce**
Create a FastAPI route that always raise an error with a large exception message (as the example below) and it will show an "event exceeded the permitted size" error on the next attempt to send events to APM server

```
@app.get('/apm_test/')
async def apm_test():
    try:
        raise Exception('test' * 1000000)
    except Exception as e:
        apm_client.capture_exception()
    return PlainTextResponse('', 200)
```

**Environment**
- OS: ubuntu 20.04
- Python version: 3.9.13
- Framework and version: fastapi==0.79.0
- APM Server version: 8.3.0
- Agent version: 6.11.0

**Comment by basepi** [comment]
This is related to #305 -- we should definitely be truncating that field but haven't been able to come up with a great agent-wide solution so we've been playing whack-a-mole when these issues pop up. Thanks for the report!

**Comment by basepi** [comment]
Fixed in #1619
